examples/application_commands/localization.py

```diff
@@ -0,0 +1,99 @@
+from nextcord import Interaction, Locale, SlashOption
+from nextcord.ext import commands
+
+TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
+BOT_TOKEN = ""  # Replace with your bot token
+
+
+bot = commands.Bot()
+
+
+@bot.slash_command(
+    name="hello",  # name of the command by default in English
+    name_localizations={
+        Locale.de: "hallo",  # localization for the name of the command in German (Deutsch)
```

Comment:
Do we really need an inline comment for almost every line?

Reply:
Is this question for other library developers or for me?

Reply:
In which lines should I remove the comments?